test/testdrive/materialized-view-refresh-options.td

```diff
@@ -383,4 +383,75 @@ true
   WHERE name = 'mv10';
 true
 
+# ----------------------------------------
+# Automated cluster scheduling for REFRESH
+# ----------------------------------------
+
+> CREATE CLUSTER scheduled_cluster (SIZE = '1', SCHEDULE = ON REFRESH, REPLICATION FACTOR = 1);
+
+# No MV yet, so the cluster should be turned off (even though it was turned on when we created it).
```

Comment:
Ah, this is exactly the kind of confusing thing I mentioned above that I think we could and should prevent by denying explicit attempts to configure the replication factor on `SCHEDULE = ON REFRESH` clusters.

Reply:
Done
416a700e3c7a01e1c76d8d00c74e718aea05e139